crates/storage/libmdbx-rs/src/transaction.rs

```diff
@@ -580,16 +583,22 @@ impl TransactionPtr {
     }
 
     fn lock(&self) -> MutexGuard<'_, ()> {
+        // Do not force capture the backtrace every time, as it can be expensive.
+        let backtrace = Backtrace::capture();
```

Comment:
but this captures the backtraces everytime?

Reply:
clarified the comment, it's not by default
<img width="971" alt="image" src="https://github.com/user-attachments/assets/e1dc81f7-5e28-4f5c-861f-1a8b2f1d66cc" />